CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ option(LIB_PROTO_MUTATOR_EXAMPLES "Enable examples building" ON)
 option(LIB_PROTO_MUTATOR_DOWNLOAD_PROTOBUF
        "Automatically download working protobuf" OFF)
 option(LIB_PROTO_MUTATOR_WITH_ASAN "Enable address sanitizer" OFF)
+option(LIB_PROTO_MUTATOR_EXAMPLES_USE_LATEST "Use latest libexpat/libxml2 for expat_example/libxml2_example (useful with OSS-Fuzz)" OFF)
```

Comment:
I don't think we need to mention particular users, like OSS-Fuzz

Reply:
@vitalybuka dropped, pushing…